chia/cmds/wallet_funcs.py

```diff
@@ -528,32 +530,34 @@ async def make_offer(
                         converted_amount = Decimal(requested_amount) / divisor
                         print(f"  - {converted_amount} {asset} ({requested_amount} mojos)")
 
-                    print()
-                    nft_confirmation = input(
-                        "Offers for NFTs will have royalties automatically added. "
-                        + "Are you sure you would like to continue? (y/n): "
-                    )
-                    if nft_confirmation not in ["y", "yes"]:
+                    if not no_confirm:
+                        print()
+                        nft_confirmation = input(
```

Comment:
the problem with solving this by introducing `no_confirm` is that you kind of just punt on the core of it. This is still not covered by tests (afaiu). Is there no way in python to write to `stdin`?

One way to make this a bit more palatable would be to wrap `input()` in our own function that we use to ask for confirmation, and it always return `True` if `no_confirm` is set. That way only that function is lacking unit tests. and maybe we can even figure out some way to test it in isolation.

Maybe en even better API is to throw if the user denies. That way, callers don't need to check the return value and exit.

Reply:
i could add this.

Reply:
added